tutorials/cp-sapcp-getstarted/cp-sapcp-getstarted.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: Get Started with the Command Line Interface for SAP Cloud Platform (sapcp CLI)
-description: Download the command line interface for SAP Cloud Platform and learn how to use it.
+title: Get Started with the SAP BTP Command Line Interface (sapcp CLI)
```

Comment:
Please use the full name "SAP Business Technology Platform" at least once, before you use the abbreviation BTP.

Reply:
hi michael, SAP BTP command line interface is the official name of the tool. It's not called SAP Business Technology Platform command line interface. Please accept it like this, the toll is always referred to like this throughout the documentation. 
Thanks a lot,
Anna